### PR TITLE
Update readme for End to End Tests and Increase code coverage to 90%

### PR DESCRIPTION
**Pull request by ashwin1111**

### CODE REVIEW

**Review by labhvam5 — APPROVED** [review]

**Comment by ashwin1111** [comment]
testing . . .

**Comment by ashwin1111** [comment]
testing 2 . . .

File: .github/workflows/e2e-test-staging.yml [hunk 1]
```diff
@@ -1,45 +1,50 @@
 name: End to End Tests on PR Approval - Staging
 
 on:
-  workflow_dispatch:
+  discussion_comment:
```
Comment (ashwin1111) [inline .github/workflows/e2e-test-staging.yml:4]: testing 3 . . .

**Comment by ashwin1111** [comment]
testing 4 . . .

**Comment by ashwin1111** [comment]
testing 5 . . .

File: .github/workflows/e2e-test-staging.yml [hunk 2]
```diff
@@ -1,45 +1,49 @@
...
```
Comment (ashwin1111) [inline .github/workflows/e2e-test-staging.yml:4]: Testing 6 . . .

**Review by Shwetabhk — APPROVED** [review]

**Comment by ashwin1111** [comment]
testing 7

File: .github/workflows/e2e-test-staging.yml [hunk 3]
```diff
@@ -1,45 +1,47 @@
 name: End to End Tests on PR Approval - Staging
 
 on:
-  workflow_dispatch:
```
Comment (ashwin1111) [inline .github/workflows/e2e-test-staging.yml:4]: testing 8

File: .github/workflows/e2e-test-staging.yml [hunk 4]
```diff
@@ -1,45 +1,47 @@
...
   pull_request_review:
     types: [submitted]
+  pull_request_review_comment:
```
Comment (ashwin1111) [inline .github/workflows/e2e-test-staging.yml:6]: test

File: .github/workflows/e2e-test-staging.yml [hunk 5]
```diff
@@ -1,45 +1,47 @@
...
+    types: [created]
 
 jobs:
```
Comment (ashwin1111) [inline .github/workflows/e2e-test-staging.yml:9]: Run e2e tests

File: .github/workflows/e2e-test-staging.yml [hunk 6]
```diff
@@ -1,45 +1,47 @@
...
   end-to-end-test:
-    if: github.event.review.state == 'approved'
+    if: (github.event_name == 'pull_request_review' && github.event.review.state == 'approved') || (github.event_name == 'pull_request_review_comment' && github.event.body == 'Run e2e tests')
```
Comment (ashwin1111) [inline .github/workflows/e2e-test-staging.yml:11]: Run e2e tests

File: .github/workflows/e2e-test-staging.yml [hunk 7]
```diff
@@ -1,45 +1,47 @@
...
-      - name: 'Upload screenshots and videos to Slack'
-        uses: trymbill/cypress-slack-video-upload-action@v1.3.0
-        if: failure()
-        with:
-          token: ${{ secrets.SLACK_TOKEN }}
-          channels: 'integration_test_runs'
-          message-text: ':pepe-dance: End to End tests (on staging) just got completed. I have placed the screenshots and videos in this thread :pepe-dance:'
-      - name: End to End Test Coverage
-        uses: fylein/comment-test-coverage@master
-        with:
```
Comment (ashwin1111) [inline .github/workflows/e2e-test-staging.yml:39]: dummy

File: .github/workflows/e2e-test-staging.yml [hunk 8]
```diff
@@ -1,45 +1,50 @@
 name: End to End Tests on PR Approval - Staging
 
 on:
-  workflow_dispatch:
```
Comment (ashwin1111) [inline .github/workflows/e2e-test-staging.yml:4]: test

File: .github/workflows/e2e-test-staging.yml [hunk 9]
```diff
@@ -1,45 +1,52 @@
...
-          tag: ${{ secrets.E2E_TESTS_ENV }}
-      - name: 'Upload screenshots and videos to Slack'
-        uses: trymbill/cypress-slack-video-upload-action@v1.3.0
-        if: failure()
-        with:
-          token: ${{ secrets.SLACK_TOKEN }}
-          channels: 'integration_test_runs'
-          message-text: ':pepe-dance: End to End tests (on staging) just got completed. I have placed the screenshots and videos in this thread :pepe-dance:'
-      - name: End to End Test Coverage
-        uses: fylein/comment-test-coverage@master
```
Comment (ashwin1111) [inline .github/workflows/e2e-test-staging.yml:38]: test

File: .github/workflows/e2e-test-staging.yml [hunk 10]
```diff
@@ -1,45 +1,52 @@
...
-          record: true
-          browser: chrome
-          tag: ${{ secrets.E2E_TESTS_ENV }}
-      - name: 'Upload screenshots and videos to Slack'
-        uses: trymbill/cypress-slack-video-upload-action@v1.3.0
-        if: failure()
-        with:
-          token: ${{ secrets.SLACK_TOKEN }}
-          channels: 'integration_test_runs'
-          message-text: ':pepe-dance: End to End tests (on staging) just got completed. I have placed the screenshots and videos in this thread :pepe-dance:'
```
Comment (ashwin1111) [inline .github/workflows/e2e-test-staging.yml:36]: ads

File: .github/workflows/e2e-test-staging.yml [hunk 11]
```diff
@@ -1,45 +1,52 @@
...
-        with:
-          record: true
-          browser: chrome
-          tag: ${{ secrets.E2E_TESTS_ENV }}
-      - name: 'Upload screenshots and videos to Slack'
-        uses: trymbill/cypress-slack-video-upload-action@v1.3.0
-        if: failure()
-        with:
-          token: ${{ secrets.SLACK_TOKEN }}
-          channels: 'integration_test_runs'
```
Comment (ashwin1111) [inline .github/workflows/e2e-test-staging.yml:35]: jkhasd

File: .github/workflows/e2e-test-staging.yml [hunk 12]
```diff
@@ -1,45 +1,52 @@
...
+      - run: echo ${{github.event.body == 'Run e2e tests'}}
+      # - name: Checkout
+      #   uses: actions/checkout@v2
+      # - name: Setup Node
+      #   uses: actions/setup-node@v1
+      #   with:
+      #     node-version: 14.18
+      # - name: Install dependencies
+      #   run: npm install --prefix cypress
+      # - name: Prepare for end to end tests
```
Comment (ashwin1111) [inline .github/workflows/e2e-test-staging.yml:29]: eee

File: .github/workflows/e2e-test-staging.yml [hunk 13]
```diff
@@ -1,45 +1,52 @@
...
-          title: End to End Test Coverage
+      - run: echo "Event - ${{toJson(github)}}"
+      - run: echo ${{github.event_name == 'pull_request_review_comment' && github.event.body == 'Run e2e tests'}}
+      - run: echo ${{github.event_name }}
+      - run: echo ${{github.event_name == 'pull_request_review_comment'}}
+      - run: echo ${{github.event.body}}
+      - run: echo ${{github.event.body == 'Run e2e tests'}}
+      # - name: Checkout
+      #   uses: actions/checkout@v2
+      # - name: Setup Node
```
Comment (ashwin1111) [inline .github/workflows/e2e-test-staging.yml:22]: asd

**Pull request merged**